packages/connector-chrome/src/chrome.ts

```diff
@@ -10,11 +10,11 @@
  * ------------------------------------------------------------------------------
  */
 
-import { Connector } from '../debugging-protocol-common/debugging-protocol-connector';
-import { ILauncher } from '../../types';
+import { Connector } from '@sonarwhal/utils-debugging-protocol-common/dist/src/debugging-protocol-connector';
```

Comment:
What about renaming `debugging-protocol-connector` to `connector`?

Having `@sonarwhal/utils-debugging-protocol-common/dist/src/connector` seems to be clear?